Makefile

```diff
@@ -10,6 +10,7 @@ export CC := /usr/local/musl/bin/musl-gcc
 GO_LDFLAGS := -X main.Version=$(HECKLER_VERSION) -extldflags=-static -linkmode=external
 export GOFLAGS := -mod=vendor -tags=static,osusergo
 export GOCACHE := $(CURDIR)/.go-build
+export GO111MODULE := on
```

Comment:
Is this necessary? As far as I can tell in go1.16 it defaults to on already.

Reply:
"go mod vendor" is expecting this "GO111MODULE" to be ON. This is for both local/mac and docker env to run make commands

Reply:
I think this was only true for go 1.15, see https://go.dev/ref/mod#mod-commands for more details